### PR TITLE
error page

### PR DESCRIPTION
**Pull request by Sapna2001**
# Related Issue
Fixes: #181

# Screenshot
Screenshot from 2021-03-21 21-55-56

### CODE REVIEW

File: templates/error/404.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% extends 'base.html' %}
```
Comment (garg3133) [inline templates/error/404.html:1]: Extend to `layout.html` instead.

File: templates/error/404.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+{% block title %} 404 Error Page {% endblock %}
```
Comment (garg3133) [inline templates/error/404.html:3]: Jagrati | 404 Page Not Found

File: templates/error/404.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+{% block content %}
+
+<style>
```
Comment (garg3133) [inline templates/error/404.html:7]: This must go into `{% block style %}`.

File: templates/error/404.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <div style="display:flex;flex-direction:row;">
+        <div style="width:45%;display:flex;justify-content:center;align-items:center;flex-direction:column;padding-top:20px;">
+            <img src="404.png">
+        </div>
+        <div style="width:45%">
+            <h1>OOPS !!!</h1>
+            <p>The page you're looking for is now beyond our reach.</p>
+        </div>
+    </div>
+    <button><a href={% url 'home:dashboard'}>Go back to Home</a></button>
```
Comment (garg3133) [inline templates/error/404.html:50]: This link is wrong.

**Review by garg3133 — COMMENTED** [review]
The overall design does not look good (like the light grey component on orange background). Can you try out something else?

**Comment by Sapna2001** [comment]
@garg3133 should I use orange and black similar to the home page

**Comment by tmo66** [comment]
Hey @Sapna2001!

- I think you can remove all the colors, from this page and let it be minimal (No background colors at all).
- Then for `The page...reach` you can use something relatable to the website, like `Ohh Seems like kids have broken this link` or `Seems like our kids misplaced the page`.
- Instead of `Go back to home` you can just write `Home`. With some shadow, border-radius, and background color `EC631B` with some opacity (if required). 
- I have a mixed opinion about the illustration, it's looking good, but the colors and the frame ruin it, so first do the changes written above and then lets see can we use a better illustration.

**Pull request closed without merging**